src/emu/services/src/window/common.cpp

```diff
@@ -148,4 +148,26 @@ namespace eka2l1::epoc {
         }
         return static_cast<TStdScanCode>(it->second);
     }
+
+    std::map<std::pair<int, int>, std::uint32_t> button_input_map = {};
```

Comment:
I dont like global modifiable variables much. Wonder if we can put this into a class/struct